How-to docs for increasing the total number of shards per node (#86214)
elastic/elasticsearch: docs/reference/troubleshooting/data/increase-shard-limit.asciidoc

[[increase-shard-limit]]

> == Maximum number of shards for an index on a single node exceeded 

Reviewer: ```suggestion
== Total number of shards for an index on a single node exceeded 
```

> == The configured maximum number of shards that can be allocated to a single node has been reached

Author: This title is a bit messy. Any better suggestions?

Reviewer 2: Perhaps "Maximum number of shards on a single node exceeded" ?

Author: Would that be clear that it's for a particular index? 

Maybe "Maximum number of shards for an index on a single node exceeded" ?

> Elasticsearch tries to take advantage of all the available resources by 
> distributing data (index shards) amongst the cluster nodes.

Reviewer 2: ```suggestion
Elasticsearch tries to take advantage of all the available resources by 
distributing data (index shards) among nodes in the cluster.
```

> Users might want to influece this data distribution by configuring the <<total-shards-per-node, index.routing.allocation.total_shards_per_node>> 

Reviewer 2: ```suggestion
Users might want to influence this data distribution by configuring the <<total-shards-per-node, index.routing.allocation.total_shards_per_node>> 
```
